generator/src/main/java/org/corfudb/generator/operations/CheckpointOperation.java

```diff
@@ -13,11 +14,17 @@
 
     public CheckpointOperation(State state) {
         super(state);
+        shortName = "Checkpoint";
     }
 
     @Override
     public void execute() {
         try {
+
+//            TODO: uncomment when verification supports it
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)